packages/api/endpoints/executions.js

```diff
@@ -79,6 +85,28 @@ async function del(req, res) {
   return res.send({ message: 'Record deleted' });
 }
 
+/**
+ * Get execution history for a single granule or multiple granules
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} the promise of express response object
+ */
+async function searchByGranules(req, res) {
+  const payload = req.body;
+  const granuleIds = await getGranuleIdsForPayload(payload);
+  const knex = await getKnexClient();
+  const executionPgModel = new ExecutionPgModel();
+
+  const executionCumulusIds = await getGranuleExecutionCumulusIds(knex, ['granule_id'], granuleIds);
```

Comment:
Is there any way that the dashboard or whatever is making this call can specify the collection name/version when making this request?

I ask because in the new PostgreSQL database, only the combination of granule_id/collection_cumulus_id makes a granule unique. `granule_id` by itself is not unique because we have a requirement to support the same granule in multiple collections (which was part of the motivation for moving to PostgreSQL). 

So I think we need to update this somehow so that the desired collection for each granule is specified so that we don't end up returning executions for the wrong granule (since it could exist for multiple collections going forward).

Reply:
Thanks for the heads up on this! I made some updates to handle an array of granule objects containing a granuleId and collectionId. The commit with all the important stuff is here: https://github.com/nasa/cumulus/pull/2324/commits/59590e4d094d6b8ea451bdabfe812d925f755772

The biggest changes were
1. Updating `getGranuleExecutionCumulusIds` to get the granule cumulus ids using the granule_id and collection_cumulus_id fields
2. Updating the elasticsearch query to return both granuleid and collection id so that we can pass that array to `getGranuleExecutionCumulusIds` in the right format.

Also updated and added some more tests. Let me know what you think!